java/hello-world/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2016 Google Inc. All Rights Reserved.
```

Comment:
Not necessary to update the copyright on existing files.

FYI: the header you should use in new files is defined at https://opensource.google.com/docs/releasing/preparing/#Apache-header (No "All rights reserved" and "Google LLC")